cockatrice/src/deckview.cpp

```diff
@@ -116,6 +117,34 @@ void DeckViewCard::mouseMoveEvent(QGraphicsSceneMouseEvent *event)
     setCursor(Qt::OpenHandCursor);
 }
 
+void DeckView::mouseDoubleClickEvent(QMouseEvent *event)
+{
+    if (event->button() == Qt::LeftButton)
+    {
+        qDebug() << "LEFT";
+        QList<MoveCard_ToZone> *result = new QList<MoveCard_ToZone>();
```

Comment:
Left it allocated on purpose until I could get it to work, then would find a safe place to de-allocate